example_test.go

```diff
@@ -60,7 +60,7 @@ var (
 		MediaType: artifactspec.MediaTypeArtifactManifest,
 		Digest:    digest.FromBytes(exampleSignatureManifest),
 		Size:      int64(len(exampleSignatureManifest))}
-	exampleSignatureManifestDescriptorArtifactspec = artifactspec.Descriptor{
+	exampleSignatureManifestDescriptorArtifactspec = ocispec.Descriptor{
```

Comment:
I think we can totally delete this variable, since after changing it to ocispec it is identical to `exampleManifestDescriptor`? We can just use `exampleManifestDescriptor`.

Reply:
nice catch!